src/main/java/bg/codexio/springframework/boot/requestcachingfilter/RequestCachingFilter.java

```diff
@@ -0,0 +1,60 @@
+package bg.codexio.springframework.boot.requestcachingfilter;
+
+import jakarta.servlet.*;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.web.util.ContentCachingRequestWrapper;
+
+import java.io.IOException;
+
+/**
+ * A servlet filter that wraps incoming {@link HttpServletRequest} instances
+ * with a {@link ContentCachingRequestWrapper} to enable multiple reads of
+ * the request body.
+ *
+ * <p>
+ * This filter is useful when the request body needs to be accessed multiple
+ * times within the processing chain, as the
+ * {@link ContentCachingRequestWrapper}
+ * caches the content for repeated reads.
+ * </p>
+ */
+public class RequestCachingFilter
+        implements Filter {
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
```

Comment:
Do we need to override them, if we use the default implementation

Reply:
done